user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Timeline_component.md

```diff
@@ -0,0 +1,130 @@
+
+     > [!NOTE]
+     > Optionally, you can use a [time range component](xref:DashboardTimeRangeFeed) to adjust and/or display the time range configured for the timeline component<!--RN 33287-->.
+     > To do so:
+     >
+     > 1. Select *Timeline # > Viewport > Timespans* in the *Feeds* section of the *Data* tab.
+     > 1. Drag it onto an empty section of the dashboard or low-code app page.
+     >
+     > When you manually adjust the time range by zooming in or out, or by moving across the timeline component, the time range displayed in the time range component will automatically be adjusted.
+
+   - *Data retrieval > Update data*: Allows updates to be enabled or disabled. This setting will enable real-time updates for all queries executed by the selected component. Disabled by default.
+
+   - *Highlight range > Use highlighting*: Determines whether an event in the timeline component is highlighted when it falls within the configured time range. When an event is highlighted, it receives a blue background color. Events that do not fall within the set time range are still visible with lowered opacity. Disabled by default<!--RN 33639-->.
+
+     ![Highlight](~/user-guide/images/Timeline_Highlight.png)<br/>*Timeline component in DataMiner 10.4.1*
+
+     To use the time range configured for another component in the dashboard or low-code app, click the ![Link to feed](~/user-guide/images/Link_to_Feed.png) icon next to the *Time range* dropdown box, and select the component from the dropdown list. Modifying the highlight time frame for this component will automatically synchronize the time range for any linked components as well.
+
+1. Fine-tune the component layout. In the *Component* > *Layout* tab, the following options are available:
+
+   - The default options available for all components. See [Customizing the component layout](xref:Customize_Component_Layout).
+
+   - *Filtering & Highlighting*: Available from DataMiner 10.1.11/10.2.0 onwards<!--RN 33276-->. Allows you to configure a number of filtering and highlighting options. However, note that the filtering options require the [Query filter component](xref:DashboardQueryFilterFeed), available from DataMiner 10.3.9/10.4.0 onwards.
+
+     - *Highlight*: When this option is enabled, the nodes that match the filter will be highlighted. Enabled by default.
+
+     - *Opacity*: When the *Highlight* option is enabled, this option will allow you to set the level of transparency of the nodes and edges that do not match the filter.
+
+       > [!NOTE]
+       > When you disable the *Highlight* option, the nodes that do not match the filter will no longer be displayed and the remaining nodes will be reorganized.
```

Comment:
*Nodes* is from the node-edge, this setting is indeed the same one as with the node-edge, but I wouldn't call it nodes in this context.